detect-nuget-inspector/detect-nuget-inspector/DependencyResolution/Project/ProjectXmlResolver.cs

```diff
@@ -82,26 +93,63 @@ public DependencyResult Process()
                     {
                         XmlAttribute include = attributes["Include"];
                         XmlAttribute version = attributes["Version"];
+                        XmlAttribute versionOverride = attributes["VersionOverride"];
                         String versionStr = null;
-                        if (version == null)
+                        String versionOverrideStr = null;
+                        
+                        if (versionOverride == null && version == null)
                         {
                             foreach (XmlNode node in package.ChildNodes)
                             {
+                                if (node.Name == "VersionOverride")
+                                {
+                                    versionOverrideStr = node.InnerXml;
+                                    break;
+                                }
                                 if (node.Name == "Version")
                                 {
                                     versionStr = node.InnerXml;
-                                    break;
                                 }
                             }
                         }
+                        else if (versionOverride != null)
```

Comment:
Does it matter if version is null?

Reply:
No it wouldn't matter if version is null we are prioritizing VersionOverride over Version because of CPM so.